__TESTS__/logEvent.controller.test.js

```diff
@@ -0,0 +1,44 @@
+const logEvent = require('../src/controllers/logEvent.controller');
+const Database = require('../src/modules/Database.module')
+const { Request } = require('jest-express/lib/request');
+const { Response } = require('jest-express/lib/response');
+
+const req = new Request();
+const res = new Response();
+let db
+
+describe('Log a new metric', () => {
+
+    beforeEach(() => {
+        db = new Database();
```

Comment:
Instantiation takes an optional integer value representing time in ms for data expiration.